### PR TITLE
completed mentors and cards styling

### PR DESCRIPTION
**Pull request by navyasingla1309**
I have made the following changes:
1. Stored mentor details in arrays in mentors page
2. rendered all elements of array through card
3. applied styling
4. Added navbar with selected link active

### CODE REVIEW

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
@navyasingla1309 the changes are not responsive for iPhone 12. I have sent you the screencast on Discord as I was unable to share it here. Please make it responsive and check it using inspect element.

**Comment by navyasingla1309** [comment]
> @navyasingla1309 the changes are not responsive for iPhone 12. I have sent you the screencast on Discord as I was unable to share it here. Please make it responsive and check it using inspect element.

Ok I will correct it.

File: pages/mentors.js [hunk 1]
```diff
@@ -1,33 +1,243 @@
...
+import Head from 'next/head'
+import Image from 'next/image'
 import Card from '../components/card'
+import Navbar from '../components/nav'
+import 'bootstrap/dist/css/bootstrap.min.css';
+
 const Mentor=()=> {
 
-    const mentors = [
+    const mentors_r1 = [
```
Comment (ShreyaPrasad1209) [inline pages/mentors.js:10]: Add "category" key in Mentors objects. Value can be anything from following:

- Competitive Programming
- Open Source
- Technical Interview Preparation

Comment (ShreyaPrasad1209) [reply pages/mentors.js:10]: Change mentors_r1 to Mentors

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]

File: pages/mentors.js (hunk 1 above)
Comment (navyasingla1309) [reply pages/mentors.js:10]: Okay, will do it.

**Review by ShreyaPrasad1209 — APPROVED** [review]
Great work @navyasingla1309 🎉 I’ll merge this PR ✅

**Pull request merged**